main.py

```diff
@@ -1 +1,35 @@
-rint("hello")
+import mysql.connector
+from mysql.connector import Error
+from fastapi import FastAPI
+from pydantic import BaseModel
+import uvicorn
+
+app = FastAPI()
+
+db_config = {
+    "host": "35.239.158.202",
+    "user": "root",
+    "password": "7$x[;*X-0qpT[J2n",
+    "database": "test",
+}
+
+connection = mysql.connector.connect(**db_config)
+
+
+class StoreItem(BaseModel):
+    naziv: str
+    cena: int
+    kolicina: int
```

Comment:
Move this and any future class to the models.py where you will store all of the classes like this